problemsets-mat223/modules/module10-exercises.tex

```diff
@@ -1,10 +1,48 @@
 \begin{exercises}
+    %Patrick:
+        %45 min
+        %60 min
+        %10 min
 	\begin{problist}
-		\prob  Example problem with parts
+	    \prob \label{Q1}Let $\mathcal S:\R^2\to\R^2$ be a transformation that doubles every vector, and let $M_{\mathcal S}$ be the corresponding matrix. Find $M_S$.
```

Comment:
Labels are global to the whole document, so a label like `Q1` is not good. change the label to `PROB-MOD10-Q1`. Also, only label problems that you refer to later on.